pkg/scheduler/metrics/metrics.go

```diff
@@ -143,7 +143,7 @@ var (
 			// Start with 0.01ms with the last bucket being [~22ms, Inf). We use a small factor (1.5)
 			// so that we have better granularity since plugin latency is very sensitive.
 			Buckets:        metrics.ExponentialBuckets(0.00001, 1.5, 20),
-			StabilityLevel: metrics.ALPHA,
+			StabilityLevel: metrics.STABLE,
```

Comment:
Can you think of a way to validate if this metric is accurate and useful?

Currently, we post at most 1000 samples every second. Since there's no particular logic to decide which samples should proceed, I fear that the information might be skewed towards certain extension point.

Reply:
It will not be skewed because we either sample the whole cycle or not. We can increase the sampling rate if we wish to make it more accurate, but in the end it depends on the number of instances we are sampling from (so number of scheduling cycles), and that depends on the cluster.

I don't think it provides useful information fleet-wide, but when debugging a specific cluster that we know uses heavily a specific feature, this metric can provide useful information on how much time that feature is adding overhead on the scheduling cycle.

Reply:
> It will not be skewed because we either sample the whole cycle or not.

Why do you say that?

Reply:
Because we don't sample based on the extension point, we either time for all extension points or for none of them.

Reply:
Also, the sampling is random

Reply:
to recap from offline discussion: There are 2 things controlling what gets recorded:
- a random sampling per scheduling cycle https://github.com/kubernetes/kubernetes/blob/6ac2d8edc8606ab387924b8b865b4a69630080e0/pkg/scheduler/scheduler.go#L441
- a buffer https://github.com/kubernetes/kubernetes/blob/6ac2d8edc8606ab387924b8b865b4a69630080e0/pkg/scheduler/framework/runtime/metrics_recorder.go#L54

My only concern is that the buffer might cause certain extension points to not be recorded because they are not called often. I think the most important one is the preemption PostFilter plugin. I would be more comfortable graduating this metric if we can ensure that the buffer is enough to fit an entire scheduling cycle for a 5k cluster.

Reply:
ok, I think we should just drop the graduation of this one for now, for the filter plugin, it would be ideal if we can measure for all nodes, not per node.

Reply:
+1 that would reduce a lot of measuring points.

Thanks!